lms/djangoapps/verify_student/tests/test_api.py

```diff
@@ -41,3 +48,77 @@ def test_send_approval(self, use_ace):
         with patch.dict(settings.VERIFY_STUDENT, {'USE_DJANGO_MAIL': use_ace}):
             send_approval_email(self.attempt)
             self._assert_verification_approved_email(self.attempt.expiration_datetime)
+
+
+@ddt.ddt
+class CreateVerificationAttempt(TestCase):
+    """
+    Test cases for the send_approval_email API method.
+    """
+
+    def setUp(self):
+        super().setUp()
+
+        self.user = UserFactory.create()
+        self.attempt = VerificationAttempt(
+            user=self.user,
+            name='Tester McTest',
+            status=VerificationAttemptStatus.created,
+            expiration_datetime=datetime(2024, 12, 31, tzinfo=timezone.utc)
+        )
+        self.attempt.save()
+
+    def test_create_verification_attempt(self):
+        expected_id = 2
+        self.assertEqual(
+            create_verification_attempt(
+                user=self.user,
+                name='Tester McTest',
+                status=VerificationAttemptStatus.created,
+                expiration_datetime=datetime(2024, 12, 31, tzinfo=timezone.utc)
+            ),
+            expected_id
+        )
+        verification_attempt = VerificationAttempt.objects.get(id=expected_id)
+
+        self.assertEqual(verification_attempt.user, self.user)
+        self.assertEqual(verification_attempt.name, 'Tester McTest')
+        self.assertEqual(verification_attempt.status, VerificationAttemptStatus.created)
+        self.assertEqual(verification_attempt.expiration_datetime, datetime(2024, 12, 31, tzinfo=timezone.utc))
+
+
+@ddt.ddt
+class UpdateVerificationStatus(TestCase):
+    """
+    Test cases for the update_verification_status API method.
+    """
+
+    def setUp(self):
+        super().setUp()
+
+        self.user= UserFactory.create()
+        self.attempt= VerificationAttempt(
+            user=self.user,
+            name='Tester McTest',
+            status=VerificationAttemptStatus.created,
+            expiration_datetime=datetime(2024, 12, 31, tzinfo=timezone.utc)
+        )
+        self.attempt.save()
+
+    @ddt.data(
+        VerificationAttemptStatus.pending,
+        VerificationAttemptStatus.approved,
+        VerificationAttemptStatus.denied,
+    )
+    def test_update_verification_status(self, to_status):
```

Comment:
When you handle the case where the `get` call fails and a `VerificationAttempt.NotFound` exception is raised, let's also test that case.